src/header-navigation/index.js

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+export {default as HeaderNavigation} from './header-navigation';
+// Styled elements
+export {
+  Root as StyledRoot,
+  NavigationItem,
```

Comment:
Should we add `Styled` to the `NavigationItem` and `NavigationList` components for consistency?

Reply:
As per @schnerd original proposal current names look more semantically right. I can rename it if needed

https://github.com/uber-web/baseui/pull/514#discussion_r234272393

Reply:
I agree with @austin94 comment, we should be consistent. So far we export any styled component with `Styled` prefix. We should be consistent.

Reply:
fixed